doc/modules/neighbors.rst

```diff
@@ -510,6 +510,66 @@ the model from 0.81 to 0.82.
     classification using nearest centroid with different shrink thresholds.
 
 
+Nearest Neighbors Transformer
```

Comment:
There are several issues with class references in this section. See https://59181-843222-gh.circle-artifacts.com/0/doc/modules/neighbors.html#nearest-neighbors-transformer. I think that with neighbors being the current module:
```
:class:`cluster.SpectralClustering` doesn't work
:class:`~sklearn.cluster.SpectralClustering` works
`SpectralClustering` works
```
but I've not checked